index.js

```diff
@@ -106,12 +106,15 @@ function applyAttributes(t, openingElement, componentName, sourceFileName, compo
     if (!node.name) return
     return node.name.name === elementAttributeName
   }) == null){
-    openingElement.node.attributes.push(
-      t.jSXAttribute(
-        t.jSXIdentifier(elementAttributeName),
-        t.stringLiteral(openingElement.node.name.name || 'unknown')
+    const name = openingElement.node.name.name || 'unknown'
+    if (ignoredElements.indexOf(name) === -1) {
```

Comment:
[Array.includes()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes) is another option. I think it's more readable to state:

```
if (ignoredElements.includes(name)) return;
openingElement.node.attributes.push(...)
```